hack/netpol/pkg/utils/k8s_util.go

```diff
@@ -98,7 +97,10 @@ func (k *Kubernetes) Probe(ns1, pod1, ns2, pod2 string, port int) (bool, error)
 	toIP := toPod.Status.PodIP
 
 	// note some versions of wget want -s for spider mode, others, -S
-	exec := []string{"wget", "--spider", "--tries", "1", "--timeout", "1", "http://" + toIP + ":" + fmt.Sprintf("%v", port)}
+	// There seems to be an issue when running Antrea in Kind where tunnel traffic is dropped at
+	// first. This leads to the first test being run consistently failing. To avoid this issue
+	// until it is resolved, we set "--tries" to 3.
+	exec := []string{"wget", "--spider", "--tries", "3", "--timeout", "0.5", "--waitretry", "0", "http://" + toIP + ":" + fmt.Sprintf("%v", port)}
```

Comment:
this is an interesting test case - should we make it part of the test matrix ??

Reply:
I replied below. It is really a bug related to running Antrea in Kind and it is orthogonal to Network Policy implementation. I will open an issue to track this today.